src/renderers/PartialViewRenderers/region/componentsInfo.js

```diff
@@ -170,65 +183,36 @@ function getFacilitiesList() {
 function rbsComponentsDisplay_onChange() {
     $('input[type=radio][name=usedComponentDisplay]').on('change', e => {
         e.preventDefault();
-        rbsComponentsDisplayer();
+        filterComponents()
     });
 
     $('input[type=radio][name=activatedComponentDisplay]').on('change', e => {
         e.preventDefault();
-        rbsComponentsDisplayer();
+        filterComponents()
     });
 }
 
-/**
- * Display component's rows based on radio button filter:
- * Used/Unused, Activated/Unactivated Components
- */
-function rbsComponentsDisplayer() {
-    let usedComponent = $('input[name=usedComponentDisplay]:checked').val();
-
-    let componentGetterEvent = 'Component:getComponentList';
-    let componentReceiverEvent = 'Component:getComponentListOK';
-    if (usedComponent === 'used') {
-        componentGetterEvent = 'Component:getUsedComponentList';
-        componentReceiverEvent = 'Component:getUsedComponentListOK';
-    } else if (usedComponent === 'unused') {
-        componentGetterEvent = 'Component:getUnusedComponentList';
-        componentReceiverEvent = 'Component:getUnusedComponentListOK';
+function filterComponents() {
+    let usedComponentInput = $('input[name=usedComponentDisplay]:checked').val();
+    let activatedComponentInput = $('input[name=activatedComponentDisplay]:checked').val();  
+    let componentList = JSON.parse(localStorage.getItem(COMPONENT_LIST))
+
+    if(usedComponentInput === USED){
```

Comment:
This code is confusing, if the used component input is used?

Reply:
If I'm not misunderstanding this, this is the filter right? How about calling it that? Similar to the other one